akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -114,7 +114,7 @@ private[remote] abstract class InboundCompression[T >: Null](
   private[this] val state: AtomicReference[InboundCompression.State[T]] = new AtomicReference(InboundCompression.State.empty)
 
   // TODO calibrate properly (h/w have direct relation to preciseness and max capacity)
-  private[this] val cms = new CountMinSketch(100, 100, System.currentTimeMillis().toInt)
+  private[this] val cms = new CountMinSketch(16, 1024, System.currentTimeMillis().toInt)
```

Comment:
Please open a ticket about that, it's a good idea but in general a good
default is what we're after for now.

-- 
Konrad `ktoso` Malawski
Akka <http://akka.io> @ Lightbend <http://lightbend.com>

On 13 September 2016 at 18:16:54, Łukasz Dubiel (notifications@github.com)
wrote:

In
akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala
<https://github.com/akka/akka/pull/21037#discussion_r78592976>:

> @@ -114,7 +114,7 @@ private[remote] abstract class InboundCompression[T >: Null](
>    private[this] val state: AtomicReference[InboundCompression.State[T]] = new AtomicReference(InboundCompression.State.empty)
>
>    // TODO calibrate properly (h/w have direct relation to preciseness and max capacity)
> -  private[this] val cms = new CountMinSketch(100, 100, System.currentTimeMillis().toInt)
> +  private[this] val cms = new CountMinSketch(16, 1024, System.currentTimeMillis().toInt)

I don't think so. Moreover it's should be configuration parameter and
depends on traffic you send. In some cases CMS with smaller width would
perform better and have accepted estimation error level.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/akka/akka/pull/21037/files/22bc850bfd2302ad16cb01fdb80ead51168d4ad2#r78592976>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAHYkzYodS-IZxNef8HQI7EwAVmKwlHrks5qpsx2gaJpZM4JUtzy>
.